website/docs/cloud-docs/run/ui.mdx

```diff
@@ -27,6 +27,8 @@ The Terraform code for a normal run always comes from version control, and is al
 
 In a workspace linked to a VCS repository, runs start automatically when you merge or commit changes to version control.
 
+-> **Note:** If you're using GitHub as your VCS provider and 300 files are reported as changed by the GitHub API when your PR is merged, Terraform Cloud will trigger all connected workspaces for that given repository. This is due to the fact that GitHub limits the number of changed files that it reports for a PR merge at 300, therefore necessitating the need for Terraform Cloud to proactively trigger workspaces to ensure that changes are not missed.
```

Comment:
According to our style guide, we try to limit the number of notes in the docs, and because this isn't something the user needs to act on, I think it works better as a paragraph.

Also took a pass at tweaking this for style; please let me know what you think!
```suggestion
If you use GitHub as your VCS provider and merge a PR changing 300 or more files, Terraform Cloud automatically triggers runs for every workspace connected to that repository. The GitHub API has a limit of 300 reported changed files for a PR merge. To address this, Terraform Cloud initiates workspace runs proactively, preventing oversight of file changes beyond this limit.
```